src/test/java/teammates/test/cases/action/InstructorFeedbackResultsDownloadActionTest.java

```diff
@@ -163,6 +176,33 @@ public void testExecuteAndPostProcess() throws Exception {
         expectedFileName = session.getCourseId() + "_" + session.getFeedbackSessionName();
         assertEquals(expectedFileName, result.getFileName());
         verifyFileContentForDownloadWithMissingResponsesHidden(result.getFileContent(), session);
+
+        ______TS("Typical case: question results downloadable");
+
+        action = getAction(paramsQuestion2);
+        result = getFileDownloadResult(action);
+
+        expectedDestination = "filedownload?" + "error=false" + "&user=idOfInstructor1OfCourse1";
+        assertEquals(expectedDestination, result.getDestinationWithParams());
+        assertFalse(result.isError);
+        assertEquals("", result.getStatusMessage());
+
+        expectedFileName = session.getCourseId() + "_" + session.getFeedbackSessionName() + "_question2";
+        assertEquals(expectedFileName, result.getFileName());
+        verifyFileContentForQuestion2Session1InCourse1(result.getFileContent(), session);
+
+        ______TS("Typical case: question results within section downloadable");
+
+        action = getAction(paramsQuestion1WithinSection);
+        result = getFileDownloadResult(action);
+
+        expectedDestination = "filedownload?" + "error=false" + "&user=idOfInstructor1OfCourse1";
+        assertEquals(expectedDestination, result.getDestinationWithParams());
+        assertFalse(result.isError);
+
+        expectedFileName = session.getCourseId() + "_" + session.getFeedbackSessionName() + "_Section 1" + "_question1";
+        assertEquals(expectedFileName, result.getFileName());
+        verifyFileContentForQuestion1Session1InCourse1WithinSection1(result.getFileContent(), session);
```

Comment:
verifyFileContentForDownload* seems to be more consistent with the current name. same for the rest.